bokeh/models/sources.py

```diff
@@ -210,6 +210,20 @@ class RemoteSource(DataSource):
 class AjaxDataSource(RemoteSource):
     method = String('POST', help="http method - GET or POST")
 
+    mode = Enum("replace", "append", help="""
+    Whether to append new data to existing data (up to ``max_size``),
+    or to replace existing data entirely.
+    """)
+    max_size = Int(help="""
+    Maximum size of the data array being kept after each pull requests.
+    Larger than that size, the data will be right shifted.
+    """)
+    if_modified = Bool(False, help="""
+    Whether to include an ``If-Modified-Since`` header in AJAX requests
+    to the server. This might allow only new data to be retrieved if
+    properly handled by the server.
```

Comment:
The second sentence reads confusingly to me. Suggest:
```
If this header is supported by the server, then only new data since the last 
request will be returned.
```